build.gradle

```diff
@@ -28,6 +29,8 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:$spotbugs_gradle_plugin_version"
         classpath "org.jetbrains.dokka:dokka-gradle-plugin:$dokka_version"
+        classpath 'com.google.dagger:hilt-android-gradle-plugin:2.38.1'
```

Comment:
this line looks unnecessary

Reply:
Ahh thx, I didn't see it was duplicated.